zipkin-lens/src/components/DependenciesPage/types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+export interface Edge {
```

Comment:
I think it's better to define / export (or not) this in the file it's used in instead of having generic `types.ts` files, alternatively `Edge.ts` would also be idiomatic but overkill I think